main/.vuepress/themeConfig/nav.js

```diff
@@ -15,6 +15,30 @@ module.exports = [
         ariaLabel: 'Agoric Dev Tools Menu',
         link: '/getting-started/',
       },
+        text: 'Before Using Agoric',
+        ariaLabel: 'Before Using Agoric Menu',
+        link: '/getting-started/before-using-agoric/',
```

Comment:
This is a blank page, but not a 404
![Screen Shot 2020-04-19 at 5 07 15 PM](https://user-images.githubusercontent.com/2441069/79703363-3e2f9b00-8260-11ea-9764-9588391637b0.png)